cmd/juju/commands/bootstrap.go

```diff
@@ -215,6 +222,29 @@ func (c *bootstrapCommand) SetFlags(f *gnuflag.FlagSet) {
 }
 
 func (c *bootstrapCommand) Init(args []string) (err error) {
+	if c.JujuDbSnapPath != "" {
+		f, err := os.Open(c.JujuDbSnapPath)
+		defer f.Close()
+		if err != nil {
+			return errors.Annotatef(err, "--juju-db-snap file unable to be opened")
+		}
+	}
+
+	// fill in JujuDbSnapAssertionsPath from the same directory as JujuDbSnapPath
+	if (c.JujuDbSnapAssertionsPath == "") && (c.JujuDbSnapPath != "") {
```

Comment:
As written, this will not test the assert file exists if the assert file is specified via the CLI rather than being guessed